### DISCUSSION TITLE
horizontal navigation reverse with android tv remote

### DISCUSSION THREAD

**Discussion by akbari6776** [react-native-tvos/react-native-tvos#412] [Q&A] [answered]
Hello
My country language is Persian, and my app is also rtl, but when I use the left button on the remote, it scrolls to the right and vice versa, between the items in the horizontal rows.

To understand the android box language, I use react-native-localize to get isRtl: true/false

When my android box language is Persian (rtl), Navigation is correct, I mean when I use left button on remote, right and vice versa, but when I set English language on my android box, navigation is reversed.

What can I do about this?

**Comment by douglowder** [comment, marked as answer]
This may be related to the issue in #262
